query-node/schemas/forumEvents.graphql

```diff
@@ -360,7 +360,7 @@ type PostReactionResultValid @variant {
 }
 
 type PostReactionResultInvalid @variant {
-  reactionId: Int!
+  reactionId: String!
```

Comment:
I don't know if Pioneer currently uses reactions, but they should be aware of this change probably
cc: @thesan